cli/compose/types/types.go

```diff
@@ -305,6 +305,7 @@ type IPAMPool struct {
 
 // VolumeConfig for a volume
 type VolumeConfig struct {
+	Name       string
```

Comment:
That link isn't going to any specific line. I don't think we should add it to the lists in `types.go`, those are for options being deprecated/removed from v2->v3.

In this case maybe just a comment in the struct  would be fine